DEVELOPMENT.md

```diff
@@ -10,21 +10,18 @@ to `Knative Serving`. Also take a look at:
 
 ## Getting started
 
-1. Create [a GitHub account](https://github.com/join)
-1. Setup [GitHub access via
-   SSH](https://help.github.com/articles/connecting-to-github-with-ssh/)
-1. Install [requirements](#requirements)
-1. [Set up a kubernetes cluster](./docs/creating-a-kubernetes-cluster.md)
-1. [Set up a docker repository you can push
-   to](./docs/setting-up-a-docker-registry.md)
-1. Set up your [shell environment](#environment-setup)
-1. [Create and checkout a repo fork](#checkout-your-fork)
-
-Once you meet these requirements, you can [start Knative Serving](#starting-knative-serving)!
+Follow the instructions below to set up your development environment.  Once you
+meet these requirements, you can
+[start Knative Serving](#starting-knative-serving)!
```

Comment:
maybe "start Knative Serving with local changes" (since you can start serving from any of the releases without these steps)?

Reply:
Slightly reworked this section, ptal.